package.json

```diff
@@ -32,7 +32,8 @@
   "author": "kiwi.com",
   "license": "MIT",
   "dependencies": {
-    "@kiwicom/orbit-design-tokens": "^0.2.5"
+    "@kiwicom/orbit-design-tokens": "^0.2.5",
+    "react-native-status-bar-height": "^2.2.0"
```

Comment:
Do we really need this library + the block in webpack.config.js ?

Reply:
- changes in webpack.config are needed for our storybook web which cannot compile without it.

- without that lib I will have to wrap component with `SafeArea` for `ios` and get `currentHeight` for `android`, that's why I think better option is to use it.